### PR TITLE
delete useles comment

### PR DESCRIPTION
**Pull request by ValeryStk**
useless comment was deleted

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ inline bool plotly_script_exists () {
 
 int main(int argc, char *argv[])
 {
-	//testing
+
```
Comment (AntonMrt) [inline main.cpp:49]: Please don't leave useless empty lines!

**Review by AntonMrt — COMMENTED** [review]
Please don't leave useless empty lines!

File: main.cpp (hunk 1 above)
Comment (ValeryStk) [reply main.cpp:49]: ok. I will do it but in future
let's automize this process with npm fromat
using c-lang style

**Review by AntonMrt — APPROVED** [review]
GOOD job! Perfect code style!

**Pull request closed without merging**